test/Microsoft.SqlTools.ServiceLayer.Test.Common/FileUtils.cs

```diff
@@ -76,5 +76,25 @@ public static string TestServerNamesDefaultDirectory
                 }
             }
         }
+
+        /// <summary>
+        /// Normalizes Windows, Unix, and mixed paths to the same slash direction, specified by <paramref name="separatorType"/>
+        /// </summary>
+        /// <param name="path"></param>
+        /// <param name="separatorType">Win32NT for \, Unix for /</param>
+        /// <returns></returns>
+        public static string NormalizePath(string path, PlatformID separatorType)
+        {
+            return separatorType switch
+            {
+                PlatformID.Win32NT => path.Contains('/')
+                                ? String.Join('\\', path.Split('/', StringSplitOptions.RemoveEmptyEntries))
+                                : path,
+                PlatformID.Unix => path.Contains('\\')
```

Comment:
What about Mac? Or does that get set as Unix?

Reply:
Mac and Linux are both Unix-based OSes, so that covers both.